src/build.props

```diff
@@ -77,12 +77,17 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Package">
+    <Title Condition=" '$(Title)' == '' ">$(AssemblyTitle)</Title>
     <Authors Condition=" '$(Authors)' == '' ">$(Company)</Authors>
-    <Owners Condition=" '$(Authors)' == '' ">$(Company),tse-securitytools</Owners>
     <PackageRequireLicenseAcceptance Condition=" '$(PackageRequireLicenseAcceptance)' == '' ">false</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl Condition=" '$(PackageLicenseUrl)' == '' ">https://github.com/Microsoft/sarif-sdk/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression Condition=" '$(PackageLicenseExpression)' == '' ">MIT</PackageLicenseExpression>
```

Comment:
>Condition [](start = 30, length = 9)

we don't need this condition or any of the others

Reply:
Removed conditions.

---
In reply to: [244648357](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648357) [](ancestors = 244648357)